samples/minimal/src/main/kotlin/io/element/android/samples/minimal/RoomListScreen.kt

```diff
@@ -80,6 +80,7 @@ class RoomListScreen(
                     initialize()
                     paginateBackwards(20, 50)
                     dispose()
+                    room.close()
```

Comment:
A bit weird to call this in the apply block on a timeline, using `use` was not possible?
But on the `minimal` sample I guess this is fine.